docker/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+NPROCS=`grep processor </proc/cpuinfo | wc -l`
+
+# Create build directory
+mkdir -p /tmp/build
+cd /tmp/build
+
+# Generate last version of parser sources from Ragel descriptors
+for i in 1 2 3 ; do ragel -G2 -o ../inc/vcf/validator_detail_v4${i}.hpp ../src/vcf/vcf_v4${i}.ragel ; done
```

Comment:
I will change the previous command to `cd /tmp` so the `..` are not necessary, and add `cd /tmp/build` after the loop, to ensure the binaries are built inside that subfolder.